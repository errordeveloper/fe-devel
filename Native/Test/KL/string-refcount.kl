operator entry() {
  String s = "foo";
  report "s.refCount = " + s.refCount;
  String t = s;
  report "s.refCount = " + s.refCount;
  t = "bar";
  report "s.refCount = " + s.refCount;
}
