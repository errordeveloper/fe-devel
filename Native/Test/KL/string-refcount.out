s.refCount = 2
s.refCount = 3
s.refCount = 2
